src/OmniSharp.LanguageServerProtocol/Handlers/OmniSharpSemanticTokensHandler.cs

```diff
@@ -22,34 +21,49 @@ public static IEnumerable<IJsonRpcHandler> Enumerate(RequestHandlers handlers)
+                [SemanticHighlightClassification.Operator] = SemanticTokenType.Operator,
+                [SemanticHighlightClassification.StringLiteral] = SemanticTokenType.String,
+                [SemanticHighlightClassification.ClassName] = SemanticTokenType.Class,
+                [SemanticHighlightClassification.StructName] = SemanticTokenType.Struct,
+                [SemanticHighlightClassification.NamespaceName] = SemanticTokenType.Namespace,
+                [SemanticHighlightClassification.EnumName] = SemanticTokenType.Enum,
+                [SemanticHighlightClassification.InterfaceName] = SemanticTokenType.Interface,
+                [SemanticHighlightClassification.TypeParameterName] = SemanticTokenType.TypeParameter,
+                [SemanticHighlightClassification.ParameterName] = SemanticTokenType.Parameter,
+                [SemanticHighlightClassification.LocalName] = SemanticTokenType.Variable,
+                [SemanticHighlightClassification.PropertyName] = SemanticTokenType.Property,
+                [SemanticHighlightClassification.MethodName] = SemanticTokenType.Method,
+                [SemanticHighlightClassification.EnumMemberName] = SemanticTokenType.EnumMember,
+                [SemanticHighlightClassification.EventName] = SemanticTokenType.Event,
+                [SemanticHighlightClassification.PreprocessorKeyword] = SemanticTokenType.Macro,
+                [SemanticHighlightClassification.LabelName] = SemanticTokenType.Label,
+            }.ToImmutableDictionary();
+
         private readonly Mef.IRequestHandler<SemanticHighlightRequest, SemanticHighlightResponse> _definitionHandler;
         private readonly DocumentSelector _documentSelector;
 
+        private static string MakeLSPCompatibleString(string str)
+            => char.ToLower(str[0]) + str.Substring(1);
+
         private static readonly ImmutableDictionary<SemanticHighlightClassification, SemanticTokenType> _tokenTypes
-            = SemanticHighlightService._classificationMap
-                .OrderBy(kvp => kvp.Value)
-                .Aggregate(
-                    new Dictionary<SemanticHighlightClassification, SemanticTokenType>(),
-                    (dictionary, kvp) =>
-                    {
-                        if (!dictionary.ContainsKey(kvp.Value))
-                            dictionary.Add(kvp.Value, new SemanticTokenType(kvp.Key));
-                        return dictionary;
-                    })
-                    .ToImmutableDictionary();
+            = System.Enum.GetValues(typeof(SemanticHighlightClassification))
+                .Cast<SemanticHighlightClassification>()
+                .ToImmutableDictionary(value => value,
+                    // Use Core LSP token types where possible
+                    value => _coreTokenMap.ContainsKey(value)
+                        ? _coreTokenMap[value]
+                        : new SemanticTokenType(MakeLSPCompatibleString(value.ToString())));
```

Comment:
Previously we had been using the Roslyn ClassificationTypeName which contained spaces. This changes us to use LSP TokenTypes when possible, otherwise SemanticHighlightClassification EnumMember names fixed up to begin with a lowercase letter.